core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -393,10 +402,10 @@ public static long getBytes(String key) {
   }
 
   /**
-   * @return a copy of the internal {@link Properties} of as an immutable map
+   * @return a view of the internal {@link Properties} of as an immutable map
    */
-  public static ImmutableMap<String, String> toMap() {
-    return Maps.fromProperties(PROPERTIES);
+  public static Map<String, String> toMap() {
```

Comment:
@gpang I checked the usage and found all the toMap is used just for getting the configurations at one time and traverse inside, so there seems no need to give a new copy.